packages/compass-e2e-tests/helpers/commands/disconnect.js

```diff
@@ -1,28 +1,28 @@
 const { delay } = require('../delay');
 const Selectors = require('../selectors');
 
-async function closeConnectionModal(app) {
-  const { client } = app;
-  await client.clickVisible(Selectors.CancelConnectionButton);
-  const connectionModalContentElement = await client.$(
+async function closeConnectionModal(compass) {
+  const { browser } = compass;
```

Comment:
Total style nit, just a bit more compact that way, feel free to ignore

```suggestion
async function closeConnectionModal({ browser }) {
```